indico/modules/categories/templates/display/event_list_fragment.html

```diff
@@ -0,0 +1,3 @@
+{% from 'categories/display/event_list.html' import event_list_block %}
+
+{{ event_list_block(events_by_month, format_event_date, is_recent, happening_now) }}
```

Comment:
No need for this template - you can call the macro directly using `get_template_module()`